cross/syncthing/Makefile

```diff
@@ -61,12 +53,11 @@ ifeq ($(strip $(STATIC_BINARIES)),1)
 CONF_ARGS += -no-upgrade
 endif
 
+#overwrite the RUN arg to the correct location
+RUN = cd $(SYNCTHING_DIR) && env $(ENV)
```

Comment:
yup, did this before deleting the other 3 lines and thought it might be less code to write but since we removed the other lines, im ok with that too